src/signals/incident/definitions/wizard-step-2-vulaan/straatverlichting-klokken.ts

```diff
@@ -53,7 +52,8 @@ const straatverlichtingKlokken = {
       wfsFilter:
         '<PropertyIsNotEqualTo><ValueReference>objecttype_omschrijving</ValueReference><Literal>Klok</Literal></PropertyIsNotEqualTo><BBOX><gml:Envelope srsName="{srsName}"><lowerCorner>{west} {south}</lowerCorner><upperCorner>{east} {north}</upperCorner></gml:Envelope></BBOX>',
       endpoint: configuration.map.layers?.verlichting,
-      maxNumberOfAssets: 1,
+      maxNumberOfAssets:
+        configuration.map.options?.maxNumberOfAssets.straatverlichtingKlokken,
```

Comment:
- [ ] if the base has options, this can be made non optional

Reply:
I think the question mark can be removed indeed.